docs/upload.rst

```diff
@@ -303,6 +303,22 @@ The auth token is sent as an ``Auth-Token`` HTTP header in the HTTP POST.
    created on stage won't work in production.
 
 
+Improving symbol upload success rate
+====================================
+
+Tecken tries to do as much as it can when handling the symbol upload request.
+Subsequent attempts will pick up where they left off--files that have been
+processed won't be reprocessed.
+
+If you find your job is getting HTTP 429s or 504s frequently or it doesn't seem
+like symbol uploads are being completed, try these tips:
```

Comment:
Should 504s be 502s, 503s and 504s per the [previous discussion and additions](https://github.com/mozilla-services/tecken/pull/2955/files#r1679272484) to the HTTP codes? IIUC the reasons for these codes could affect both the download and upload APIs.